tools/RAiDER/delay.py

```diff
@@ -399,14 +402,17 @@ def _build_cube_ray(
                 else:
                     pts = np.stack(pts, axis=-1)
 
+                ## adjust the lowest level by ~ 1 mm to account for transformation error
+                if (pts[:, :, -1] < ht).all():
+                    pts[:, :, -1] = ht
+
```

Comment:
What happens if this doesn't happen? It's not clear why you need this.

Reply:
So there was a bug in raider previously?

Reply:
was it not working before?

Reply:
Please see the comment on the change log which explains this fix.